### DISCUSSION TITLE
On-premise Deployment Considerations

### DISCUSSION THREAD

**Discussion by uci-oit-or** [usnistgov/NEMO#123] [General]
Hello @rptmat57, we're planning on rolling this out on-premise soon. :smiley: 

This is a great piece of software, and is a good fit for us.

Our experience so far has been great, and we were able to make it work for us. Documentation has been good as well.

Before we flip the switch, anything we should be aware of? Any longer term plans from NIST, requirement guidelines for contributions, etc. outside of the wiki guidelines?
Thanks,
-Chris Price
Office of Research OIT
UC Irvine

**Comment by rptmat57** [comment]
Hello Chris,
Thank you for the support, I am glad NEMO is a good fit for your facility!

In terms of future plans and support:

1. As far as I am aware, NIST plans on keeping NEMO's development active for the foreseeable future since it is used pretty heavily in multiple facilities. 
2. The company I work for, [Prometheus Computing](https://nemo.prometheuscomputing.com/) is also very involved in helping facilities with NEMO in areas where NIST's support is limited (installation, integration, maintenance, custom development etc.).
3. The number of facilities using NEMO is constantly growing and this means there are more and more support, ideas, solutions and developers available out there.

For contributions, I would recommend starting a discussion or issue first, so we can figure out if we would be open to PRs for the requested feature, and so we can figure out potential limits and solutions.
In the event PRs are not an option, I would recommend looking at some of the already developed [NEMO open-source plugins on GitLab](https://gitlab.com/nemo-community).

This repository as well as the [mailing list ](https://groups.google.com/a/list.nist.gov/d/forum/nemo) are actively monitored so feel free to reach out with anything NEMO related!

Thanks again!

**Reply by uci-oit-or** [reply]
Awesome, thank you for the information. Do you think you'll have another conference for NEMO? Seems like you had the first one in 2021 but I didn't see another for this year.

**Reply by rptmat57** [reply]
We are talking about it. We were at the UGIM in June and had a presentation there (the original NEMO conference was in lieu of the UGIM which was postponed due to the pandemic).
One of the option we have been looking at was to organize a NEMO Conference every other year on odd years (UGIM happens on even years) to avoid too much repetition/duplication and to have enough time to organize it. If we end up doing that it would happen Spring/Summer 2023. But nothing has been confirmed yet.

**Reply by uci-oit-or** [reply]
okay great! thanks, I'll keep in touch with the mailing list
